ifbcat_api/model/training.py

```diff
@@ -148,8 +148,8 @@ def create_new_event(self, start_date, end_date):
         event = Event.objects.create(**event_attrs)
 
         if start_date:
-            d, created = EventDate.objects.get_or_create(dateStart=start_date, dateEnd=end_date)
-            event.dates.add(d)
+            event_attrs['start_date'] = start_date
```

Comment:
Ce bloque doit être avant 135 : le bloque for à partir de 135 va lire le contenu de event_attrs pour créer l'event. Tel que c'est écrit là, event_attrs est changé après la création de l'event, et les dates ne sont donc pas prises en comptes